### PR TITLE
Adding Norb Pool

### PR DESCRIPTION
**Pull request by nova4005**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -2115,5 +2115,20 @@
     "telegram":"t0m3kf",
     "operatorName":"",
     "github":"t0m3kf"
+  },
+   {
+    "ticker":"NORB",
+    "poolId":"71b00c23502112d4c634e325f28c9ade2a4ba4ec251677d306ae6b7b",
+    "social":{
+      "facebook":"",
+      "twitter":"NorbPool",
```
Comment (leo42) [inline registry.json:2124]: Hey. how about we get some action on twitter. Just to make sure a real person is behind it.

**Review by leo42 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (nova4005) [reply registry.json:2124]: Hey, I have made a post and added some information for location and bio. I will continue to add new posts. Thank you!

Comment (nova4005) [reply registry.json:2124]: I just released my first video on my YouTube channel if you want to check it out. I am looking forward to joining the alliance 😀!

**Review by Bez625 — CHANGES_REQUESTED** [review]
Please can you remove the changes to adapools.json? This file is updated automatically

**Comment by nova4005** [comment]
Hey, Sorry about that. I have removed the other commits

**Review by Bez625 — APPROVED** [review]

**Pull request merged**